CustomConfigurationProvider/Controllers/CustomConfigurationController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using YCompany.Configurations;
+
+namespace CustomConfigurationProvider.Controllers
+{
```

Comment:
controller is not requried in this project. this project will be referrenced in other project which will actually use the configuration provider.